aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -521,6 +526,7 @@ module aptos_framework::voting {
     public entry fun test_voting_failed_early(aptos_framework: signer, governance: signer) acquires VotingForum {
         account::create_account_for_test(@aptos_framework);
         timestamp::set_time_has_started_for_testing(&aptos_framework);
+        status::set_genesis_end_for_test(&aptos_framework);
```

Comment:
I wonder if we should group both of the timestamp and status calls into a test function in genesis. Something like setup_network(). It does suck a bit that every test has to call it explicitly. May be this is something we can improve the unit test runner to call by default in the future.

Reply:
I removed it for the same reason above. We don't do `assert_operating` at runtime currently.